bootcd/rpms/genesis_scripts/src/run-genesis-bootloader

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
Should drop a `set -e` here in case something blows up, so we bail early and the operator doesn't have to sift thru loads of failing output from subsequent commands